src/main/java/org/cirdles/topsoil/app/RowFactory.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
This needs a license. You can go to project properties and specify `apache20` as the license and "CIRDLES" as the organization to generate it automatically.